### ISSUE TITLE
Save and show all irrelevant

### ISSUE THREAD

**Issue by catap** [CitationGecko/gecko-react#45]
Right now if you save session and load it at another browser, you will loose all irrelevant marked fields.

Is it possible to save it?

Also, it will be lovley if it's shown all irrelevant articles to easy undo some mistakes.

**Comment by barney-walker** [comment]
Yep, should be pretty easy to save the 'irrelevant' annotation as part of the bibtex. I will include this in the next update!
